module/User/src/Service/AclService.php

```diff
@@ -52,34 +52,16 @@ protected function createAcl()
         $this->acl->addRole(new Role('tueguest'), 'guest');
         $this->acl->addRole(new Role('user'), 'tueguest');
         $this->acl->addrole(new Role('apiuser'), 'guest');
-        $this->acl->addrole(new Role('sosuser'), 'apiuser');
         $this->acl->addrole(new Role('active_member'), 'user');
         $this->acl->addrole(new Role('company_admin'), 'active_member');
         $this->acl->addRole(new Role('admin'));
         $this->acl->addRole(new Role('photo_guest'), 'guest');
 
-        $user = $this->getIdentity();
-
-        // add user to registry
-        if ($user instanceof User) {
-            $roles = $user->getRoleNames();
-            // if the user has no roles, add the 'user' role by default
-            if (empty($roles)) {
-                $roles = ['user'];
-            }
-
-            if (count($user->getMember()->getCurrentOrganInstallations()) > 0) {
-                $roles[] = 'active_member';
-            }
-
-            $this->acl->addRole($user, $roles);
-        }
-
         // admins are allowed to do everything
         $this->acl->allow('admin');
 
         // board members also are admins
-        $this->acl->allow('user', null, null, new IsBoardMember());
+        $this->acl->allow('active_member', null, null, new IsBoardMember());
```

Comment:
This does not work with my comment regarding the `company_admin` check.

We already discussed the part below, but I like to mention it here for documentation purposes.

Furthermore, I disagree that we can make this assumption. Yes, in practice this will always be the case, but we cannot guarantee it. I would rather see a separate `board` role which is applied using the same logic but as part of the `User->getRoleId()`.

That way it is clearly that a board member has a clear and predefined hierarchy & role (that of a board member) and not an active member with extended capabilities.

Reply:
I'll come up with a way that applies this assertion to all roles instead.